go.mod

```diff
@@ -404,6 +404,7 @@ require (
 	github.com/hashicorp/mdns v1.0.4 // indirect
 	github.com/hashicorp/net-rpc-msgpackrpc/v2 v2.0.0 // indirect
 	github.com/hashicorp/serf v0.10.1 // indirect
+	github.com/hashicorp/vault-licensing v1.6.0 // indirect
```

Comment:
I think we probably want to add this to the `vault_ent` go.mod over on enterprise  instead of here.